Remotion/Core/Reflection.CodeGeneration.TypePipe/RemotionPipelineRegistry.cs

```diff
@@ -37,7 +37,8 @@ public class RemotionPipelineRegistry : DefaultPipelineRegistry
     private static IPipeline CreateDefaultPipeline (IEnumerable<IParticipant> defaultPipelineParticipants)
     {
       var remotionPipelineFactory = new RemotionPipelineFactory();
-      var settings = new AppConfigBasedSettingsProvider().GetSettings();
+      // We don't resolve the settings from IoC here as this the "remotion-default-pipeline" and customization is not needed at this point
+      var settings = new PipelineSettingsProvider().GetSettings();
```

Comment:
I don't understand. The original AppConfigBaseSettingsProvider read its settings from the config file, which could be customized. Why don't we need this any more?